Prime.cpp

```diff
@@ -0,0 +1,25 @@
+//This Code all prints all the prime number till input n 
+
+#include <bits/stdc++.h>
+using namespace std;
+int main()
+{
+	int n,i,j,count=0; 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `c/cpp spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp25unf0ja/Prime.cpp
+++ b/tmp/tmp25unf0ja/Prime.cpp
@@ -4,7 +4,7 @@
 using namespace std;
 int main()
 {
-	int n,i,j,count=0; 
+	int n,i,j,count=0;
 	cin>>n;
 	for(i=2;i<n;i++)
 	{
```